src/Halogen/HTML/Events.purs

```diff
@@ -100,6 +108,20 @@ onScroll = handler (EventType "scroll")
 onChange :: forall r i. (Event -> Maybe i) -> IProp (onChange :: Event | r) i
 onChange = handler ET.change
 
+onFileUpload :: forall r i t. Unfoldable t
+             => (t File -> Maybe i) -> IProp (onChange :: Event | r) i
+onFileUpload f = handler ET.change $
+  Event.target >=>
+  HTMLInputElement.fromEventTarget >=>
+  (HTMLInputElement.files >>> unsafePerformEffect) >=>
+  (items >>> pure) >=>
+  f
+  where -- Should be removed with https://github.com/purescript-web/purescript-web-file/pull/8
```

Comment:
Looking good now! Just one thing, is this comment still right? As that PR was merged/released yesterday: https://github.com/purescript-web/purescript-web-file/releases/tag/v2.2.0

Reply:
I don't see purescript-web-file in the bower.json to update the version.
How do I use the new version?

Reply:
That just means it's a transitive dependency, if you install the dependencies afresh it should include it - but there's no harm in adding an explicit dependency of `web-file` ^2.2.0 either, that's probably better even.

Reply:
Done